spirv_glsl.hpp

```diff
@@ -99,6 +99,10 @@ class CompilerGLSL : public Compiler
 		// In non-Vulkan GLSL, emit push constant blocks as UBOs rather than plain uniforms.
 		bool emit_push_constant_as_uniform_buffer = false;
 
+		// Enable using buffer blocks. If disabled, uniform/shader storage/push constant blocks are never
+		// emitted as buffer blocks, regardless the GLSL version.
+		bool enable_buffer_blocks = true;
```

Comment:
Shader storage buffers must be banned if using this option, since it's not possible to use legacy uniforms with that feature. There needs to be some kind of check for that, or SSBOs should ignore this option, and the option name must be clearer, like "enable_uniform_buffer_blocks" or something.

Reply:
Ok, renamed to emit_uniform_buffer_as_plain_uniforms with false by default (this also follows the pattern set by emit_push_constant_as_uniform_buffer better), and it is now ignored for pc and ssb blocks.